btax/run_btax.py

```diff
@@ -35,9 +35,9 @@ def run_btax(user_params):
 		:rtype: DataFrame
 	"""
 	# break out the asset data by entity type (c corp, s corp, sole proprietorships, and partners)
-	entity_dfs = pull_soi_data()
+	#entity_dfs = pull_soi_data()
```

Comment:
@bfgard , FYI, I've commented this out for now since we aren't doing overall METRs (where we'd including land and inventories) and because we'll start off reporting industry splits only by those the BEA uses.  When we look at METRs on fixed assets by asset or industry, I don't think we need the SOI data since we are assuming the mix of fixed assets is the same across tax treatment within a given industry.  When we add those other forms of capital (or allow for other levels of industry detail), we'll rely on this function again to help attribute assets across tax treatment.